bin/apisix

```diff
@@ -286,7 +286,18 @@ http {
 
     {% if enable_admin and port_admin then %}
     server {
+        {%if https_admin then%}
```

Comment:
I think we need to update the `conf/config.yaml` file, add new field `https_admin`